README.md

```diff
@@ -58,14 +69,30 @@ Secret name must be `postgres-certificates-tls-secret`
 kubectl create secret generic postgres-certificates-tls-secret --from-file=$PGSERVER_CRT_FILE --from-file=$PGSERVER_KEY_FILE --from-file=$PGCA_CERTS_FILE -n qa-qkm
 ```
 
-### client certificate
+# 4 Manifests
 
-fill the `postgresql` part in the file below.
+Manifests values must be passed to the chart using the `B64_MANIFESTS` env var. This env var is required.
 
-[here](./values/qkm.yaml.gotmpl)
+Using the provided manifest sample in the manifests dir you can build this value using the following command
 
-format is base64 pem encoded file for each value
+```
+export B64_MANIFESTS=$(envsubst < manifests/manifests.yaml | base64)
+```
+
+Please note that you will need `envsubst` tool installed and that you should fill related env vars accordingly to fit your needs.
```

Comment:
we could add `envsubst` in a Requirements section at the top of the readme

Reply:
Ok, I will add a section